pkg/pkg.bzl

```diff
@@ -382,7 +401,10 @@ pkg_deb_impl = rule(
     implementation = _pkg_deb_impl,
     attrs = {
         "data": attr.label(mandatory = True, allow_single_file = tar_filetype),
-        "package": attr.string(mandatory = True),
+        "package": attr.string(
+            doc = "Package name",
+            mandatory = True,
+        ),
```

Comment:
I'm not a fan of this particular attribute name.  It seems like this might be confusing when compared with `package_file_name` below?

For the experimental `pkg_rpm` rule, we went with `rpm_name`, which was used in place of `name` when provided.  How about doing something similar here, like `package_name`?  It could be added to the common attributes documentation list.

This can be deferred, but should be assigned a task so that deb/rpm/$SYSTEM_PACKAGE_MANAGER have consistent naming schemes for metadata like this.

Reply:
I'm deferring this because it has been in use so long. package_name is probably better.